### ISSUE TITLE
High memory usage + flickering 

### ISSUE THREAD

**Issue by redsigma** [AikoMidori/steam-dark-mode#29]
If you visit this profile:

https://steamcommunity.com/profiles/76561198032710895 

Steam memory reaches 2.5-3gb  . With default steam it reaches max 1.3 gb

The fix:

1. Search for `.responsive_page_content{overflow-x:hidden!important}` and remove it (to be precise remove the `overflow-x:hidden!important`)
2. Search for `.profile_content.has_profile_background` and remove the `border-radius:var(--border1)!important;`

I wrote more detail about this issue here:
https://github.com/purogamer/Fluent-for-Steam/issues/15#issuecomment-1044183162

TL:DR 
It seems that `border-radius` causes some strange bugs to happen
